homeassistant/components/reolink/strings.json

```diff
@@ -37,5 +37,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "https_webhook": {
+      "title": "Reolink webhook URL uses HTTPS (SSL)",
+      "description": "Reolink products can not push motion events to an HTTPS address (SSL), please configure a (local) HTTP address under `Home Assistant URL` in the [network settings](https://my.home-assistant.io/redirect/network/). The current webhook address is: `{webhook_address}`"
```

Comment:
We should not add URLs in here (problematic for translations and updating them in the future), use a placeholder instead.

The last sentence is a bit odd, as it internal context that a user might not be aware of. I wonder if that adds value or not, or we can just drop it?

Reply:
`The current webhook address is: {webhook_address}` was indeed intended because a user might not be aware of the internal URL homeassistant is using if it has been set up automatically (although I believe it will then always be a HTTP address not a HTTPS address) or if it was configured a long time ago.

I am fine with removing that last part.
Schould I remove it?

Reply:
The URL has been replaced by a placeholder.

Reply:
@frenck I have changed it to only show the base_url and changed "webhook address" to "(local) address"